### PR TITLE
[Jenkins-55787] Switch labels from entry to checkbox

### PR DESCRIPTION
**Pull request by jsoref**
# Description

See [JENKINS-55787](https://issues.jenkins-ci.org/browse/JENKINS-55787).

Additionally make the UI more consistent by removing trailing periods:
image

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: src/main/resources/org/thoughtslive/jenkins/plugins/hubot/config/HubotJobProperty/config.jelly [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    <f:entry field="enableNotifications" title="Enable Notifications">
-      <f:checkbox/>
+    <f:entry field="enableNotifications">
+      <f:checkbox title="Enable Notifications"/>
     </f:entry>
     <f:entry field="siteNames" title="Site Name">
       <f:select/>
     </f:entry>
   </f:section>
-</j:jelly>
```
Comment (nrayapati) [inline src/main/resources/org/thoughtslive/jenkins/plugins/hubot/config/HubotJobProperty/config.jelly]: Can you leave these new lines at the of these files please.

Comment (jsoref) [reply src/main/resources/org/thoughtslive/jenkins/plugins/hubot/config/HubotJobProperty/config.jelly]: That message is actually saying you don't have new lines at the end of your files now and that this change is putting them in.

I can with significant effort avoid adding those end-of-line at end-of-file markers, but each editor will try to put them in, so it's a losing battle and a waste of effort.

Comment (nrayapati) [reply src/main/resources/org/thoughtslive/jenkins/plugins/hubot/config/HubotJobProperty/config.jelly]: I want to add those new lines those got removed with this change.

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Comment by nrayapati** [comment]
Thank you for the change @jsoref

**Pull request merged**